src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -7684,6 +7683,23 @@ public static string NormalizeToJsonNetFormat(this string json)
                         sb.Append("u002f");
                     }
                 }
+                // Convert > to \u003e
+                else if (json[i] == '>')
```

Comment:
This is also problematic for `WriteStringsWithRelaxedEscaping` as mentioned [above](https://github.com/dotnet/runtime/pull/35042/files#r409273109).